launcher/game/updater.rpy

```diff
@@ -71,9 +71,29 @@ init python:
     _("Nightly")
     _("The bleeding edge of Ren'Py development. This may have the latest features, or might not run at all.")
 
+    # Newly added functions. Signed, MPDS
+    # Update: GET OUT OF MY HEAD
 
+    filter_keywords=set()
+    
+    def display_keywords():
+        return "Search phrases:\n"+"\n".join(filter_keywords)
```

Comment:
Removing the first `\n` works better than spacing it out as such with the UI aesthetic.